mindmap/mindmap.py

```diff
@@ -589,7 +589,7 @@ def remove_grade(self, data, _suffix="") -> dict:
         # Lazy import: import here to avoid app not ready errors
         from submissions.api import reset_score  # pylint: disable=import-outside-toplevel
 
-        require(self.is_instructor())
+        require(self.is_instructor() or self.is_course_staff(self.get_current_user()))
```

Comment:
Can you make this a property?

Reply:
Yeah. But I'll need to do some refactoring first

Reply:
Done @Ian2012. Can you check again?